### PR TITLE
Prepare for release 1.1.0

### PR DESCRIPTION
**Pull request by kozlovic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,10 +50,10 @@ To start the NATS server with default settings (and no authentication or cluster
 
 ```sh
 > ./gnatsd
-[1764] 2018/02/28 14:29:45.033940 [INF] Starting nats-server version 1.0.6
-[1764] 2018/02/28 14:29:45.034017 [INF] Git commit [not set]
-[1764] 2018/02/28 14:29:45.034255 [INF] Listening for client connections on 0.0.0.0:4222
-[1764] 2018/02/28 14:29:45.034262 [INF] Server is ready
+[7499] 2018/03/23 11:46:15.239889 [INF] Starting nats-server version 1.1.0
+[7499] 2018/03/23 11:46:15.239964 [INF] Git commit [not set]
```
Comment (derekcollison) [inline README.md]: I assume this will be set on a release build correct?

File: README.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ The NATS server uses a [text based protocol](http://nats.io/documentation/intern
 Trying 107.170.221.32...
 Connected to demo.nats.io.
 Escape character is '^]'.
-INFO {"server_id":"mxyLRV3o96bPBziXld35wt","version":"1.0.6","go":"go1.8.4","host":"0.0.0.0","port":4222, ...}
+INFO {"server_id":"JUOufISZqT09TSan4iBHCJ","version":"1.0.6","git_commit":"","go":"go1.9.4","host":"0.0.0.0","port":4222,"auth_required":false,"ssl_required":false,"tls_required":false,"tls_verify":false,"max_payload":1048576}
```
Comment (derekcollison) [inline README.md]: I wonder if we should deprecate ssl_required.

File: README.md [hunk 3]
```diff
@@ -679,19 +679,20 @@ Examples using the test certificates which are self signed for localhost and 127
...
 Notice that the log  indicates that the client connections will be required to use TLS.  If you run the server in Debug mode with `-D` or `-DV`, the logs will show the cipher suite selection for each connected client.
 
 ```
-[72167] 2017/06/30 13:08:42.658129 [DBG] ::1:63035 - cid:4 - Starting TLS client connection handshake
-[72167] 2017/06/30 13:08:42.674096 [DBG] ::1:63035 - cid:4 - TLS handshake complete
-[72167] 2017/06/30 13:08:42.674121 [DBG] ::1:63035 - cid:4 - TLS version 1.2, cipher suite TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+[7883] 2018/03/23 11:49:50.603842 [DBG] ::1:58352 - cid:1 - Client connection created
+[7883] 2018/03/23 11:49:50.603908 [DBG] ::1:58352 - cid:1 - Starting TLS client connection handshake
+[7883] 2018/03/23 11:49:50.619115 [DBG] ::1:58352 - cid:1 - TLS handshake complete
+[7883] 2018/03/23 11:49:50.619127 [DBG] ::1:58352 - cid:1 - TLS version 1.2, cipher suite TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
```
Comment (derekcollison) [inline README.md]: Where is support for 1.3? That coming in golang 1.10?

Comment (wallyqs) [reply README.md]: Seems most likely to be included in either Go 1.11 or Go 1.12  https://github.com/golang/go/issues/9671

File: README.md (hunk 2 above)
Comment (kozlovic) [reply README.md]: You mean removed from INFO, but keep in structs that we have? If we remove from structs, people embedding the NATS Server will have a problem...

Comment (kozlovic) [reply README.md]: humm... so we have a struct in client_test.go which has `TLSRequired` but uses `json:"ssl_required"`, so this one would be easy to fix. The other is in `client.go` and although this is a non exported struct we return it through `ClientAuthentication.GetOpts()`.. that is weird. That struct has only `SslRequired`.

Comment (derekcollison) [reply README.md]: See if we can clean it up. I think those embedding NATS server will be ok on a recompile unless they explicitly set sslRequired.

**Pull request merged**